xradar/io/export/cfradial1.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
+# Copyright (c) 2022, openradar developers.
+# Distributed under the MIT License. See LICENSE for more info.
+
+"""
+
+CfRadial1 output
+================
+
+This sub-module contains the writer for export of CfRadial1-based radar
+data.
+
+Author: @syedhamidali.
```

Comment:
Remove, your contributions are tracked in git anyway and are added as contributor.